flytekit/core/base_task.py

```diff
@@ -462,6 +463,7 @@ def __init__(
         environment: Optional[Dict[str, str]] = None,
         disable_deck: Optional[bool] = None,
         enable_deck: Optional[bool] = None,
+        decks: Optional[Tuple[str, ...]] = ("Source Code", "Dependencies"),
```

Comment:
@novahow I know we've been moving back and forth about this, but using capitalized placeholders seem weird to me.

At this point, I'm okay with using the enums: `DeckField.SOURCE_CODE`, etc.